src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -148,6 +148,10 @@ export const ChooseApiPage = () => {
     );
   });
 
+  const debouncedSearch = debounce((searchString: string) => {
```

Comment:
The reason it's done this way is because the function that is to be debounced needs to be bounded in order to fire with the right parameter - it needs the context surrounding it, and the easiest way to provide that is by binding it with an arrow function. It wouldn't have to be this way if the debounce-function was implemented right here in this file, but since I've made it into a reusable function, the context for the handleSearch-function needs to be provided so that it can be fired from the different file/context that the debounce-function's code resides in.